ChangeLog.md

```diff
@@ -20,6 +20,11 @@ See docs/process.md for more on how version tagging works.
 
 2.0.24
 ------
+- Interally the `libgl` family of libraries was renamed to `libGL`.  This should
+  not be a visible change as `-lGL` as always been the way to include this
+  library.  Anyone who uses `embuilder` to build this library specifiecally
+  would need to change from `embuilder build libgl..` to `embuilder build
+  libGL..` (#14351).
 - Support `--preload-file` in Node.js. (#11785)
```

Comment:
Reading this, I wonder if maybe it's worth adding backwards compatibility support in embuilder. That is, to map the old name to the new name there, so that this is truly an NFC change that users cannot notice. However, I'm not sure how many people use embuilder directly. Up to you.

Reply:
I think the number of users who use embuilder in this way today is probably vanishingly small.  I'd like to see more folks doing it but I think pretty much everyone relies on the auto-building.

Plus, `embuilder --help` will tell always tell you what your options are.